support/latest/policies/containerization.md

```diff
@@ -1,21 +1,19 @@
 ---
-title: Containerization Support Policy
+title: Deployment and Containerization Support Policy
 ---
 
-Since the release of Alfresco Content Services 6.0 and Alfresco Process Services 1.6, Alfresco supports the deployment of our products using both the traditional deployment methods and our new containerized deployment mechanisms.
-
-Alfresco supports deployment using the open industry-standard technology Docker for containerization and Kubernetes for orchestration. We provide reference Docker images and a reference orchestrated deployment using Kubernetes and Helm. These deployment templates are there to enable customers to understand the best practices for deployment of our platform in the cloud.
+![Containerized deployment]({% link support/images/deployment-support.png %})
 
-As with traditional on premise deployment methods, customers may wish to customize or build upon the technology and ideas within our deployment templates. The deployment templates provide a starting point for customers to simplify the creation of deployments unique to their own needs and skill sets.
+The above illustration shows four deployment options. Alfresco creates and supports only the software represented as App1, App2, and App3. Alfresco doesn’t create or support anything else in the diagram - that’s considered to be “infrastructure”. 
```

Comment:
LGTM - my only suggestion is to add an introduction to the page rather than starting with the image. Another option is to move this paragraph to the top of the page, and then follow it with the diagram (adjusting the wording as needed).

Reply:
This suggestion makes sense, but I'm struggling to come up with a good way of addressing it. Moving the first sentence above the diagram is possible, but doesn't seem to add anything really. Moving the entire first paragraph makes it feel odd - the second sentence only makes sense once the diagram has been viewed. The idea of an introduction is probably best, but I've failed to come up with something that adds rather than detracts from the rest of it. 

Because of this, I've chosen not to do anything. I haven't ignored the suggestion :)

Reply:
No problem - thanks for the consideration :-)